packages/core/src/SlideBox/Arrow.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes, { InferProps } from 'prop-types'
+import { ChevronRight, ChevronLeft } from 'pcln-icons'
+import { ARROW_SIZES } from './Arrow.styles'
+import { Relative } from '../Relative'
+import { Button } from '../Button'
+
+const { bool, string, func } = PropTypes
+
+export const arrowPropTypes = {
+  isLeft: bool,
+  arrowButtonVariation: string,
+  size: string,
+  leftArrowClick: func,
+  rightArrowClick: func,
+  leftDisabled: bool,
+  rightDisabled: bool,
+}
```

Comment:
n-b we're trying to start using interfaces instead of `InferProps`

Reply:
Updated